### PR TITLE
Open Replay Folder in Replay Menu

### PR DESCRIPTION
**Pull request by nevespt**

### CODE REVIEW

File: Assets/Script/Replays/ReplayContainer.cs [hunk 1]
```diff
@@ -353,5 +353,10 @@ private static ReplayFrame CreateReplayFrame(int id, BasePlayer player, out int
 
             return frame;
         }
+
+        public static void OpenReplayDirectory()
+        {
+            System.Diagnostics.Process.Start(ReplayDirectory);
```
Comment (EliteAsian123) [inline Assets/Script/Replays/ReplayContainer.cs:359]: Please use `FileExplorerHelper.OpenFolder` instead, as this does not work on all platforms.

File: Assets/Script/Settings/Customization/CustomContentManager.cs [hunk 2]
```diff
@@ -22,5 +22,10 @@ public static void LoadContent()
             ColorProfiles.LoadFiles();
             CameraSettings.LoadFiles();
         }
+
+        public static void OpenCustomizationDirectory()
+        {
+            System.Diagnostics.Process.Start(CustomizationDirectory);
```
Comment (EliteAsian123) [inline Assets/Script/Settings/Customization/CustomContentManager.cs:28]: This is unused.

File: Assets/Script/Menu/Replays/ReplaysMenu.cs [hunk 3]
```diff
@@ -84,6 +84,7 @@ private void OnEnable()
             {
                 new NavigationScheme.Entry(MenuAction.Green, "Select", () => CurrentReplay?.Confirm()),
                 new NavigationScheme.Entry(MenuAction.Red, "Back", () => MenuManager.Instance.PopMenu()),
+                new NavigationScheme.Entry(MenuAction.Select, "Open Folder", () => ReplayContainer.OpenReplayDirectory()),
```
Comment (EliteAsian123) [inline Assets/Script/Menu/Replays/ReplaysMenu.cs:87]: This should also probably be a UI button on the header (similar to the “Add Profile”) button on the profiles menu. I'll merge this PR without it, but that might make this better.

**Review by EliteAsian123 — CHANGES_REQUESTED** [review]

File: Assets/Script/Menu/Replays/ReplaysMenu.cs (hunk 3 above)
Comment (nevespt) [reply Assets/Script/Menu/Replays/ReplaysMenu.cs:87]: we can ask Kadu about the ui

**Comment by nevespt** [comment]
@EliteAsian123 this should work now, i hope

**Review by EliteAsian123 — APPROVED** [review]

**Comment by EliteAsian123** [comment]
Sorry for the wait!

**Pull request merged**